contrib/dist/linux/README

```diff
@@ -40,42 +40,50 @@ The folowing arguments could be used to affect script behaviour.
 Please, do NOT set the same settings with parameters and config vars.
 
 -b
-   If you specify this option, all-in-one binary RPM would be created
-   instead of source RPM (SRPM). This option is necessary for other
-   parameters to work, as they won't be influence SRPM package.
+   If you specify this option, only all-in-one binary RPM would be build.
+   By default, only source RPM (SRPM) is build. This option is necessary
+   for all other parameters to work, as they work only with binary RPM.
 
--n <name>
-   This option will change the name of the produced RPM to the <name>.
+-n name
+   This option will change the name of the produced RPM to the "name".
    It is useful to use with "-o" and "-m" options if you want to have
-   multiple OpenMPI versions installed simultaneously in the same
-   enviroment.
+   multiple Open MPI versions installed simultaneously in the same
+   enviroment. Requires use of option "-b".
 
 -o
    With this option the install path of the binary RPM will be changed
-   to /opt/<name>/<version>
+   to /opt/_NAME_/_VERSION_. Requires use of option "-b".
 
 -m
-   RPM would also install modulefiles to location specified in RPM
-   spec file.
+   This option causes the RPM to also install modulefiles
+   to the location specified in specfile. Requires use of option "-b".
 
--d
-   Script will also build debuginfo RPM.
+-i
+   Also build a debuginfo RPM. By default, debuginfo RPM is not build.
+   Requires use of option "-b".
+
+-f lf_location
+   Include support for Libfabric. "lf_location" is path of the Libfabric
+   header files. Requires use of option "-b".
 
--u
-   Build RPM with usNIC support. It will include all the recommended
-   compile options for usNIC as well.
+-t tm_location
+   Include support for Torque/PBS Pro. "tm_location" is path of the
+   Torque/PBS Pro header files. Requires use of option "-b".
+
+-d
+   Build with debugging support. By default,
+   RPM is build without debugging support.
 
--f <lf_location>
-   Include support for LibFabric. <lf_location> is path of the LibFabric
-   header files.
+-c parameter
+   Add custom configure parameter.
 
--t <tm_location>
-   Include support for Torque. <tm_location> is path of the Torque
-   header files.
+-r parameter
+   Add custom RPM build parameter.
 
 -s
-   If specified, script will try to unpack openmpi.spec file
-   from tarball
+   If specified, script will try to unpack openmpi.spec file from
+   tarball specified on the command line. By default, script is
+   looking for specfile in current directory. Requires use of option "-b".
```

Comment:
If specified, the script will try to unpack the openmpi.spec file from the tarball specified on the command line.  By default, the script will look for the specfile in the current directory.